### PR TITLE
ENCD-6121-add-screen-tiling-reference

### PR DESCRIPTION
**Pull request by khzlin**

### CODE REVIEW

File: src/encoded/schemas/reference.json [hunk 1]
```diff
@@ -204,6 +213,16 @@
                     "transcription factors"
                 ]
             }
+        },
+        "crispr_screen_tiling": {
+            "title": "CRISPR screen tiling",
+            "description": "CRISPR screen guide RNA design modalities. Peak tiling = all possible grna in a selection of peaks. Full tiling = all possible grna across an entire region, including regulatory elements and non regulatory elements. Sparse peaks = a small set of grna against a selection of peaks",
```
Comment (bonitalam) [inline src/encoded/schemas/reference.json]: I think anything after modalities can be added as a comment and moved from the description. I would also have "grna" be "gRNA" instead.

File: src/encoded/schemas/reference.json [hunk 2]
```diff
@@ -204,6 +213,16 @@
...
+            "type": "string",
+            "enum": [
```
Comment (bonitalam) [inline src/encoded/schemas/reference.json:222]: Can you put these enums in alphabetical order for consistency?

File: src/encoded/static/components/search.js [hunk 3]
```diff
@@ -651,6 +651,8 @@ globals.listingViews.register(Annotation, 'Annotation');
 
 const DatasetComponent = ({ context: result, auditIndicators, auditDetail }, reactContext) => {
     const haveFileSet = result['@type'].indexOf('FileSet') >= 0;
+    const fileSetType = result['@type'][0];
```
Comment (bonitalam) [inline src/encoded/static/components/search.js]: I think the UI part of the ticket should be for Functional Characterization Experiment and Functional Characterization Series. I think Idan will clarify more on the Jira ticket.

**Review by bonitalam — CHANGES_REQUESTED** [review]

File: src/encoded/schemas/reference.json (hunk 1 above)
Comment (khzlin) [reply src/encoded/schemas/reference.json]: ok, fixed

File: src/encoded/static/components/search.js (hunk 3 above)
Comment (khzlin) [reply src/encoded/static/components/search.js]: second commit titled "UI new specs" contains additional code for new specs.

**Review by bonitalam — DISMISSED** [review]

**Review by bonitalam — APPROVED** [review]

**Pull request merged**